### DISCUSSION TITLE
Request for Korean Language Support in Piper TTS

### DISCUSSION THREAD

**Discussion by redrocki** [rhasspy/piper#680] [General]
Hello Piper Team,

I discovered Piper TTS through the Calibre book management program, which supports both book management and reading functionalities. Since the integration of Piper TTS into Calibre for Windows users, I have found that my reading experience has significantly improved. It allows me to understand books faster and in a more comfortable manner.

However, I noticed that Korean is not yet supported among the available languages in Piper TTS. I also saw that Chinese has recently been added, which shows the continuous effort to expand language support.

I would like to inquire whether there are any plans to include Korean language support in the near future. If so, could you please provide an estimated timeline for its release? Korean language support would greatly benefit users like me and further expand the reach of Piper TTS.

Thank you for considering this request, and I look forward to your response.

**Comment by martinarisk** [comment]
Here is such a model: https://huggingface.co/neurlang/piper-onnx-kss-korean
